release-notes.txt

```diff
@@ -1,3 +1,528 @@
+* sys/shell: fix getopt() support (#20209)
++ sys/busy_wait: add busy wait helper (#20241)
+
+Networking
+----------
+
+* gnrc: make message queues static (#19998)
++ nanocoap: implement coap_find_uri_query() (#20273)
++ pkg/lwip: add support for slipdev (#20022)
+* coap: create typedef for CoAP methods (#20003)
+* gcoap_dns: Housekeeping with regards to draft status (#20015)
+* gnrc/ipv6/nib: allow for predictable static link-local addresses (#20224)
+* gnrc/nib: allow prefix to be on-link without being used for address (#20243)
+* nanocoap: fix incorrect uses of coap_get_code() (#20028)
+* nanocoap_sock: implement FETCH methods (#20238)
+* print: Missing size_t print format specifiers (#20249)
+
+Packages
+--------
+
+* Bumb nanopb to 0.4.8 (#20132)
+* mjson: Initial include of package (#20129)
+* pkg/ccn-lite: Bump version (#20248)
+* pkg/flashdb: bump to 2.1.0 (#20155)
+* pkg/nanocbor: Bump to latest commit (#19975)
+* pkg/tinydtls: bump version (#20159)
+* pkg: micropython: bump version (#20208)
+* pkg: 64-bit support preparations (#20271)
+
+Boards
+------
+
++ boards/feather-nrf52840-sense: Add bootloader info (#20034)
++ boards/gd32vf103c-start: new board (#20270)
++ boards/generic-cc2538-cc2592-dk: add board support (#20226)
++ boards/nucleo-l4r5zi : add PWM configuration (#20036)
++ boards/nucleo-l552ze: add PWM configuration (#20032)
++ boards/p-nucleo-wb55: add PWM configuration (#20120)
++ boards: add support for Adafruit Feather nRF52840 Sense (#20027)
++ drivers/include/periph: add FREQM peripheral driver (#20020)
++ drivers/ws281x: Add gpio_ll and timer based driver (#19891)
++ pyterm: add native support (#20172)
++ sys/shell: add firmware version to version cmd (#20244)
+* AVR-8: Implement Power Management (#19784)
+* boards/nrf52dk: Minimal Arduino pinout support (#20286)
+* boards/same54-xpro: configure pins for UART HW flow control &
+  ethernet LED (#20183)
+* boards: remove `extern mtd_dev_t *mtd<n>` declarations from board
+  definitions (#20104)
+* makefiles/tests/tests.inc.mk: use native term for tests (#20215)
+* treewise: typo "or RAM" (#20219)
+
+CPU
+---
+
+* cpu/msp430: rework MSP430 x1xx periph drivers (#19835)
+* cpu/nrf5x: clean up periph_uart (#20102)
+* cpu/stm32: FMC used for low-level LCD parallel interface (#19943)
++ cpu/esp32/periph: add low-level SDMMC peripheral driver for ESP32 and
+  ESP32-S3 (#19786)
++ cpu/msp430: improve periph_timer (#20160)
++ cpu/stm32/periph: add low-level SDMMC peripheral driver (#19540)
++ cpu/atmega_common: implement periph_timer_query_freqs (#20142)
++ cpu/cc26xx_cc13xx: implement periph_timer_query_freqs (#20143)
++ cpu/kinetis: implement periph_timer_query_freqs (#20144)
++ cpu/nrf5x_common: implement periph_timer_query_freqs (#20145)
++ cpu/qn908x: implement periph_timer_query_freqs (#20146)
++ cpu/sam0_common: implement periph_timer_query_freqs (#20147)
++ cpu/stm32: add ADCAL operation (#20044)
++ cpu/stm32: implement periph_timer_query_freqs (#20148)
```

Comment:
Theses `periph_timer_query_freqs` changes may also be something to highlight.

Reply:
They're also something that could be grouped in the change log.